src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PathSeqPipelineSpark.java

```diff
@@ -56,6 +56,13 @@
             minValue = 100)
     public int readsPerPartition = 5000;
 
+    @Argument(doc = "Number of reads per partition for output.",
+            fullName = "readsPerPartitionOutput",
+            optional = true,
+            minValue = 100,
+            minRecommendedValue = 100000)
+    public int readsPerPartitionOutput = 1000000;
```

Comment:
It seems like maybe this could be handled through the "numReducers" argument instead of adding a new argument?

Reply:
Not sure if I understood what @lbergelson is suggesting, but there's the `getRecommendedNumReducers()` method in `GATKSparkTool`.

But Mark's application is facing a non traditional-GATK domain so I trust his judgement.

Reply:
I just wasn't sure if he knew about that argument.  If it doesn't work for his purposes that's fine, but if he just hadn't seen it maybe it's worth looking at since it does something very similar.

Reply:
I was unsure whether to use numReducers but on second look I can't find anywhere it's used other than for a sort when writing a single BAM. If that's the case I might as well just use numReducers instead of creating a new option here.